pom.xml

```diff
@@ -399,5 +406,7 @@
         <testng.version>6.9.10</testng.version>
         <jacoco.version>0.7.9</jacoco.version>
         <maven.surefire.plugin.version>2.18.1</maven.surefire.plugin.version>
+        <identity.extension.utils>1.0.8</identity.extension.utils>
+        <identity.extension.utils.import.version.range>[1.0.8,2.0.0)</identity.extension.utils.import.version.range>
     </properties>
 </project>
```

Comment:
add a blank line

Reply:
Fixed in https://github.com/wso2-extensions/identity-outbound-auth-duo/pull/22/commits/1f21dce91d956f7f59b80e69032e5d854eb604c3